src/components/accordion/accordion.e2e.ts

```diff
@@ -47,23 +47,58 @@ describe("calcite-accordion", () => {
-    const icon2 = await page.find(`calcite-accordion-item[id='2'] >>> .${CSS.iconStart}`);
-    const icon3 = await page.find(`calcite-accordion-item[id='3'] >>> .${CSS.iconStart}`);
-    expect(icon1).not.toBe(null);
-    expect(icon2).toBe(null);
-    expect(icon3).not.toBe(null);
+  describe("icon behavior", () => {
+    let page: E2EPage;
+    const scale = { l: "l", m: "m", s: "s" };
+    const accordionItemContent = html`<calcite-accordion-item icon-start="car" id="1"></calcite-accordion-item>
+      <calcite-accordion-item id="2"></calcite-accordion-item>
+      <calcite-accordion-item icon-start="car" id="3"></calcite-accordion-item>`;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(html`<calcite-accordion scale="l"> ${accordionItemContent} </calcite-accordion>`);
+      await page.waitForChanges();
+    });
+
+    it("renders icon if requested", async () => {
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      const icon2: E2EElement = await page.find(`calcite-accordion-item[id='2'] >>> .${CSS.iconStart}`);
+      const icon3: E2EElement = await page.find(`calcite-accordion-item[id='3'] >>> .${CSS.iconStart}`);
+      expect(icon1).not.toBe(null);
+      expect(icon2).toBe(null);
+      expect(icon3).not.toBe(null);
+    });
+
+    it("renders m scale icon for l scale accordion-item", async () => {
+      const item1: E2EElement = await page.find(`calcite-accordion-item[id='1']`);
+      expect(await item1.getProperty("scale")).toBe(scale.l);
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      expect(await icon1.getProperty("scale")).toBe(scale.m);
+    });
+
+    it("renders corresponding scale on accordion-item when parent scale changes, icon scale not affected", async () => {
+      const accordion: E2EElement = await page.find("calcite-accordion");
+      await accordion.setProperty("scale", scale.s);
+      await page.waitForChanges();
+
+      const item1: E2EElement = await page.find(`calcite-accordion-item[id='1']`);
+      expect(await item1.getProperty("scale")).toEqual(scale.s);
+
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      expect(await icon1.getProperty("scale")).toEqual(scale.m);
+    });
+
+    it("renders expected scale icon on child when scale is set on child level (no parent override)", async () => {
+      const accordion: E2EElement = await page.find("calcite-accordion");
+      await accordion.getProperty("scale", scale.l);
+
+      const item1: E2EElement = await page.find(`calcite-accordion-item[id='1']`);
+      await item1.setProperty("scale", scale.m);
+      await page.waitForChanges();
+
+      expect(await accordion.getProperty("scale")).toEqual(scale.l);
+      expect(await item1.getProperty("scale")).toEqual(scale.m);
```

Comment:
Don't we want the scale of `calcite-accordion` and `calcite-accordion-item` to match in this case?

Reply:
> Don't we want the scale of `calcite-accordion` and `calcite-accordion-item` to match in this case - shouldn't let folks set scale "m" on the child if the parent is "l". (assuming scale on `calcite-accordion-item` is internal property)

I agree it seems odd to allow this, but if you think about it, this pattern breaks component autonomy.  When a component defines its own `scale` prop, I as a developer expect it to honor whatever value I'm explicitly setting it to, excepting the default fallback value.  If a parent value comes in and overrides that value I've explicitly set on the child, the property pattern on that child component no longer behaves according to standard convention.

Reply:
Yeah, although if the property is internal and is set by / managed by the parent maybe that wouldn't be expected. Though to follow through on that the scale probably shouldn't have the default to "m" until it receives the scale from `calcite-accordion`.

Reply:
This same principle applies to object inheritance in programming.  If I have Object A that defines a `foo` property, an Object B that extends Object A, I don't expect setting the `foo` property on Object A to override what I've explicitly set it to on Object B.  I expect the lowest level of specificity on that property to win above all others.

Reply:
Right - but its not a public property (or, is proposed to not be an undocumented, internal property) - so the test case of it being set in that way should not be valid?

Reply:
> Yeah, although if the property is internal and is set by / managed by the parent maybe I wouldn't expect to be able to set this at all, as its undocumented. Though to follow through on that the scale probably shouldn't have the default to "m" until it receives the scale from `calcite-accordion`.

For sure.  I think with a component like `accordion`, its children items are so tightly interwoven with the parent that this pattern does make sense where you would explicitly control children properties like `scale` from the parent level.  It gets trickier when you have child components that can live standalone by themselves like `radio-button` and `tile-select` where they need to work on their own regardless if their corresponding parent wrapper component is being used.

> Right - but its not a public property (or, is proposed to not be an undocumented, internal property) - so the test case of it being set in that way should not be valid?

Agreed.  I think if we mark these properties as internal, and call out in our documentation that modifying internal properties is not supported, then this pattern can be adopted for certain parent/child component systems.